examples/dreambooth/train_dreambooth.py

```diff
@@ -289,6 +294,14 @@ def parse_args(input_args=None):
     parser.add_argument(
         "--enable_xformers_memory_efficient_attention", action="store_true", help="Whether or not to use xformers."
     )
+    parser.add_argument("--samples_per_checkpoint", type=int, default=0, help="Whether or not to save samples for every checkpoint specified by --checkpointing_steps.")
+    parser.add_argument("--sample_steps", type=int, default=40, help="Number of steps for generating sample images.")
```

Comment:
Can we instead maybe use: https://github.com/huggingface/diffusers/blob/d75ad93ca73e00bc59d980004c4fbe798f598498/examples/dreambooth/train_dreambooth_lora.py#L171 ?